MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -16,25 +16,19 @@ final class MapViewModel: ObservableObject {
 
     @Published var userPosition: MapCameraPosition = .userLocation(fallback: .automatic)
 
+    private let farmerService = FarmerService()
```

Comment:
Your service is still not injected. I suggest you to use the init of your viewModel to make it easily testable.

Reply:
I made the changes in the last commit